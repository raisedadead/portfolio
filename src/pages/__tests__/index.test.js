import IndexMarkup from '../../pages/index';
import React from 'react';
import renderer from 'react-test-renderer';

describe(`HomePage`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<IndexMarkup />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
