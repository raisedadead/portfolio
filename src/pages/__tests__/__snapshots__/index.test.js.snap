// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<div
  className="css-11inlkn elghiqq0"
>
  <div>
    <header>
      <nav
        className="css-14zqg3i e1befqft0"
      >
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={0}
          href="https://git.raisedadead.com/resume/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span>
                  résumé
                </span>
              </span>
            </span>
          </span>
        </a>
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={1}
          href="https://github.com/raisedadead/ama/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span
                  className="css-1h0pl34 ew65k9p0"
                  id="tooltip-1"
                >
                  <span
                    className="css-pusqig ew65k9p3"
                  >
                    <span
                      aria-describedby="tooltip-1-content"
                      aria-owns="tooltip-1-content"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onFocus={[Function]}
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      ama
                    </span>
                  </span>
                </span>
              </span>
            </span>
          </span>
        </a>
      </nav>
    </header>
    <div>
      <main>
        <section
          className="css-moci5h elghiqq0"
        >
          <div
            className="e7r6m60 css-esc6uk elghiqq0"
          >
            <div
              className="e7r6m61 css-l8vtm2 elghiqq0"
            >
              <h1
                className="css-o2sjh0 e19le4zx0"
              >
                mrugesh mohapatra
              </h1>
            </div>
            <div
              className="e7r6m61 css-l8vtm2 elghiqq0"
            >
              <h2
                className="css-ov77vs e19le4zx0"
              >
                developer. music addict. open source enthusiast. noob photographer.
              </h2>
            </div>
            <div
              className="e7r6m61 css-l8vtm2 elghiqq0"
            >
              <h4
                className="css-105a4qn e19le4zx0"
              >
                Developer Advocate at freeCodeCamp.org
              </h4>
            </div>
          </div>
        </section>
      </main>
    </div>
  </div>
  <footer>
    <div
      className="css-moci5h elghiqq0"
    >
      <nav
        className="css-1ogyrb7 e1befqft0"
      >
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={0}
          href="https://twitter.com/raisedadead"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span>
                  Twitter
                </span>
              </span>
            </span>
          </span>
        </a>
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={1}
          href="https://github.com/raisedadead"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span>
                  GitHub
                </span>
              </span>
            </span>
          </span>
        </a>
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={2}
          href="https://blog.raisedadead.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span>
                  Medium
                </span>
              </span>
            </span>
          </span>
        </a>
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={3}
          href="https://freeCodeCamp.org/raisedadead"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span>
                  freeCodeCamp
                </span>
              </span>
            </span>
          </span>
        </a>
        <a
          className="e1befqft1 css-p91rce e70ypka2"
          data-index={4}
          href="https://linkedin.com/in/mrugeshm"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="css-1xssqnp e70ypka1"
          >
            <span
              className="css-1sp7cuy e70ypka0"
            >
              <span
                className="css-gh8y4m e1i5p1da0"
              >
                <span>
                  LinkedIn
                </span>
              </span>
            </span>
          </span>
        </a>
      </nav>
      <h6
        className="css-1xj32bh e19le4zx0"
      >
         
        © 
        2019
         Mrugesh Mohapatra.
      </h6>
    </div>
  </footer>
</div>
`;
